movement/watch_faces/complication/randonaut_face.c

```diff
@@ -0,0 +1,411 @@
+                    watch_display_string("0", _get_pseudo_entropy(10));
+                    watch_display_string("11", _get_pseudo_entropy(10));
+                    watch_display_string("00", _get_pseudo_entropy(10));
+                    delay_ms(50);
+                    watch_display_string(" ", _get_pseudo_entropy(10));
+                    watch_display_string(" ", _get_pseudo_entropy(10));
+                    watch_display_string(" ", _get_pseudo_entropy(10));
+                    watch_display_string(" ", _get_pseudo_entropy(10));
+                }
+            _generate_blindspot(state);
+            watch_clear_display();
+            state->face.mode = 2; // point
+            state->face.location_format = 1; // distance
+            watch_display_string("RA   Found", 0);
+            delay_ms(500);
+            sprintf(buf, "RA   Found");
+            break;
+        case 2: //point
+            switch ( state->face.location_format ) {
+                case 0:
+                    sprintf(buf, "RA   Point");
+                    break;
+                case 1: // distance to point
+                    watch_clear_display();
+                    sprintf(buf, "DI m %d", state->point.distance );
+                    break;
+                case 2: // bearing relative to point
+                    watch_clear_display();
+                    sprintf(buf, "BE # %d", state->point.bearing );
+                    break;
+                case 3: // latitude DD._____
+                    sprintf(state->scratchpad, "%07d", abs(state->point.latitude));
+                    sprintf(buf, "LA #%c %c%c  ", state->point.latitude < 0 ? '-' : '+', state->scratchpad[0], state->scratchpad[1]);
+                    break;
+                case 4: // latitude __.DDDDD
+                    sprintf(buf, "LA , %c%c%c%c%c", state->scratchpad[2], state->scratchpad[3],state->scratchpad[4], state->scratchpad[5],state->scratchpad[6]);
+                    break;
+                case 5: // longitude DD._____
+                    sprintf(state->scratchpad, "%08d", abs(state->point.longitude));
+                    sprintf(buf, "LO #%c%c%c%c  ", state->point.longitude < 0 ? '-' : '+',state->scratchpad[0], state->scratchpad[1], state->scratchpad[2]);
+                    break;
+                case 6: // longitude __.DDDDD
+                    sprintf(buf, "LO , %c%c%c%c%c", state->scratchpad[3], state->scratchpad[4],state->scratchpad[5], state->scratchpad[6],state->scratchpad[7]);
+                    break;
+            }
+            break;
+        case 3: // setup radius
+            watch_set_colon();
+            if ( state->radius < 10000 )
+                sprintf(buf, "RA m %d ", state->radius);
+            else
+                sprintf(buf, "RA m%d ", state->radius);
+            break;
+        case 4: // setup RNG
+            sprintf(buf, "RN G %s ", state->chance ? "Chnce" : (state->quantum ? "True" : "Psudo"), state->radius);
```

Comment:
There's only one format specifier here and two arguments — did you mean to have a second format specifier, or to remove the `state->radius`?

Reply:
good catch! yes that's a leftover from an earlier version of the setting. will fix it.